modules/specialized-guides/pages/salt/salt-formula-liberate.adoc

```diff
@@ -8,14 +8,11 @@ With this formula the conversion will take place during the client onboarding on
 
 
 
-== Install the formula
-
-Install the formula with:
-
-----
-transactional-update pkg in liberate-formula
-----
+[NOTE]
+====
 
+The Liberate Formula is already preinstalled on the server container.
```

Comment:
The Liberate Formula comes preinstalled within the server container.

Reply:
Thank you, fixed.